common/models/templates/src/trie.ts

```diff
@@ -160,10 +160,12 @@ export class TrieTraversal implements LexiconTraversal {
     }
   }
 
-  *children(): Generator<{char: USVString, traversal: () => LexiconTraversal}> {
+  children(): {char: USVString, p: number, traversal: () => LexiconTraversal}[] {
```

Comment:
This is probably also faster, if we are not leveraging generator for partial results!